aslam_cv/aslam_cv_backend/include/aslam/backend/CameraDesignVariable.hpp

```diff
@@ -42,6 +47,14 @@ namespace aslam {
 
 		boost::shared_ptr<camera_t> camera() { return _camera; }
 
+		/// \brief Get the keypoint time as an expression. If the shutter
+		///        parameters are being estimated, this will be hooked up
+		///        to the camera design variable
+		backend::ScalarExpression keypointTime(const aslam::Time & imageStamp, const Eigen::VectorXd & y);
```

Comment:
could it make sense to move this functionality to a subclass specific to rolling shutter cameras? or would it complicate things unnecessarily?